Dockerfile.centos8.dapper

```diff
@@ -1,5 +1,9 @@
 FROM centos:8
 
+# CentOS 8 has reached EOL: https://www.centos.org/centos-linux-eol/
```

Comment:
I merged the other PR so this will need a rebase.

Reply:
Did you have a look at #12 which includes the less hacky fix?

Anyway, this branch was rebased.